### PR TITLE
add view AssetMarketApiView, update urls, write tests

### PR DESCRIPTION
**Pull request by Lex-23**
- update model YahooMarket
- write view AssetMarketApiView
- update urls
- write tests for new endpoint

### CODE REVIEW

**Comment by Lex-23** [comment]
[Write MarketClient](https://trello.com/c/AEeD8oPC/10-write-marketclient)

File: cryptostock/market/models.py [hunk 1]
```diff
@@ -52,7 +52,10 @@ def get_assets(self):
         return self.get_assets_from_yahoo()
 
     def get_asset(self, name):
-        asset = [asset for asset in self.get_assets() if asset["name"] == name][0]
+        try:
+            asset = [asset for asset in self.get_assets() if asset["name"] == name][0]
+        except IndexError:
+            return None
```
Comment (vadim-denisov) [inline cryptostock/market/models.py:58]:
```suggestion
        assets = (asset for asset in self.get_assets() if asset["name"] == name)
        asset = next(assets, None)
```

File: cryptostock/market/views.py [hunk 2]
```diff
@@ -10,3 +11,19 @@ def get(self, request, name, format=None):
         validate_is_broker(request)
         market = get_object_or_404(queryset=Market.objects.all(), name=name)
         return Response(market.client.get_assets())
+
+
+class AssetMarketApiView(APIView):
+    def get(self, request, format=None, **kwargs):
```
Comment (vadim-denisov) [inline cryptostock/market/views.py:17]:
```suggestion
    def get(self, request, format=None, market_name, asset_name):
```

File: cryptostock/market/models.py (hunk 1 above)
Comment (Lex-23) [reply cryptostock/market/models.py:58]: Looks fine, got it

File: cryptostock/market/views.py (hunk 2 above)
Comment (Lex-23) [reply cryptostock/market/views.py:17]: got it

**Review by vadim-denisov — APPROVED** [review]

**Pull request merged**